jax/_src/lax/linalg.py

```diff
@@ -1778,11 +1780,15 @@ def _tridiagonal_solve_jax(dl, d, du, b, **kw):
 
   return x_[::-1]
 
-
 mlir.register_lowering(tridiagonal_solve_p, mlir.lower_fun(
     _tridiagonal_solve_jax, multiple_results=False))
 
 
+def _tridiagonal_solve_vmap(batched_args, batch_dims, **_):
```

Comment:
The primitive itself doesn't support batch dimensions, mainly because it exists as a primitive so that we can target backend-specific kernels. Those kernels aren't batched (AFAIK) so let's just call into the HLO implementation here!